homeassistant/components/guardian/sensor.py

```diff
@@ -49,7 +49,7 @@ class ValveControllerSensorDescription(
         name="Battery",
         device_class=SensorDeviceClass.BATTERY,
         entity_category=EntityCategory.DIAGNOSTIC,
-        native_unit_of_measurement=PERCENTAGE,
+        native_unit_of_measurement=ELECTRIC_POTENTIAL_VOLT,
```

Comment:
The battery sensor device class always uses percent as unit.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes